### PR TITLE
Add training materials

### PR DESCRIPTION
**Pull request by donnayoyo**
Add 'Tutorials' session for exercises

### CODE REVIEW

**Comment by donnayoyo** [comment]
Since I don't have right to add 'reviwer', hopefully, the comments will direct you to review the code:
@mborgerson
@myguitar
@huang-jin-intel

Thanks!

File: source/index.rst [hunk 1]
```diff
@@ -24,4 +24,5 @@ Welcome to |LPN| (version |version|) documentation!
    security/index
    how-tos/index
    tools/index
+   tutorial/index
```
Comment (huang-jin-intel) [inline source/index.rst]: change 'tutorial' to 'tutorials'

File: source/tutorial/ex1.rst [hunk 2]
```diff
@@ -0,0 +1,42 @@
+.. _Exercise 1:
+
+Exercise 1 \\- \ Build Bootloader for QEMU
+---------------
```
Comment (huang-jin-intel) [inline source/tutorial/ex1.rst]: The coding style is to make the underline as long as the header above

File: source/tutorial/ex1.rst [hunk 3]
```diff
@@ -0,0 +1,42 @@
+.. _Exercise 1:
+
+Exercise 1 \\- \ Build Bootloader for QEMU
```
Comment (huang-jin-intel) [inline source/tutorial/ex1.rst]: Change to "Build QEMU"

File: source/tutorial/ex2.rst [hunk 4]
```diff
@@ -0,0 +1,40 @@
+.. _Exercise 2:
+
+Exercise 2 \\- \ Run Bootloader on QEMU
```
Comment (huang-jin-intel) [inline source/tutorial/ex2.rst]: Run |SPN| on QEMU Emulator

File: source/tutorial/ex3.rst [hunk 5]
```diff
@@ -0,0 +1,61 @@
+.. _Exercise 3:
+
+Exercise 3 \\- \ SBL Shell/CLI
```
Comment (huang-jin-intel) [inline source/tutorial/ex3.rst]: |SPN|

File: source/tutorial/ex7.rst [hunk 6]
```diff
@@ -0,0 +1,62 @@
+.. _Exercise 7:
+
+Exercise 7 \\- \ Feature Customization - Boot Order
+----------------
+
+.. note::
+  **In this exercise, we'll learn how to change boot order.**
```
Comment (huang-jin-intel) [inline source/tutorial/ex7.rst]: ... how to change boot order from source code.

File: source/tutorial/ex7.rst [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+  - Look through the boot log on Command Prompt and locate string ``Getting boot image from...`` to determine  the 1st boot device::
+  
+        python BuildLoader.py build qemu
+
+
+  .. image:: /images/ex7-1.jpg
+    :alt: Compile completed
+    :align: center
+  
+---------
```
Comment (huang-jin-intel) [inline source/tutorial/ex7.rst]: remove spacer

**Pull request merged**